app/models/project.rb

```diff
@@ -79,4 +83,8 @@ def self.extract_tags(tags)
     end
     tags_dict
   end
+
+  def has_languages
```

Comment:
Inconsistent indentation detected.<br>Rename `has_languages` to `has_languages?`.